shapely/_geometry.py

```diff
@@ -337,18 +351,19 @@ def get_point(geometry, index, **kwargs):
 
     Examples
     --------
-    >>> line = Geometry("LINESTRING (0 0, 1 1, 2 2, 3 3)")
+    >>> from shapely import LinearRing, LineString, MultiPoint, Point
+    >>> line = LineString(((0, 0), (1, 1), (2, 2), (3, 3)))
     >>> get_point(line, 1)
-    <pygeos.Geometry POINT (1 1)>
+    <shapely.Point POINT (1 1)>
     >>> get_point(line, -2)
-    <pygeos.Geometry POINT (2 2)>
+    <shapely.Point POINT (2 2)>
     >>> get_point(line, [0, 3]).tolist()
-    [<pygeos.Geometry POINT (0 0)>, <pygeos.Geometry POINT (3 3)>]
-    >>> get_point(Geometry("LINEARRING (0 0, 1 1, 2 2, 0 0)"), 1)
-    <pygeos.Geometry POINT (1 1)>
-    >>> get_point(Geometry("MULTIPOINT (0 0, 1 1, 2 2, 3 3)"), 1) is None
+    [<shapely.Point POINT (0 0)>, <shapely.Point POINT (3 3)>]
+    >>> get_point(LinearRing(((0, 0), (1, 1), (2, 2), (0, 0))), 1)
+    <shapely.Point POINT (1 1)>
+    >>> get_point(MultiPoint(((0, 0), (1, 1), (2, 2), (3, 3))), 1) is None
```

Comment:
```suggestion
    [<shapely.Point POINT (0 0)>, <shapely.Point POINT (3 3)>]

    The function works the same for LinearRing input:

    >>> get_point(LinearRing(((0, 0), (1, 1), (2, 2), (0, 0))), 1)
    <shapely.Point POINT (1 1)

    For non-linear input geometries, it returns None:

    >>> get_point(MultiPoint(((0, 0), (1, 1), (2, 2), (3, 3))), 1) is None
```

In general, this kind of divisions of the example blob would make them more readable, I think. But that's probably a bit out of scope for this PR, so feel free to defer.

Reply:
I'll add this one here but we should certainly do a more comprehensive pass at this in another PR.

Reply:
Scratch that; I'm updating these now.  All these lines were touched in this PR anyway, best to have them clean before this gets merged.